docker/mongo/docker-compose.yml

```diff
@@ -30,7 +30,11 @@
 version: '3.3'
 
 services:
+
   pygeoapi:
+    env_file:
```

Comment:
@KilimAnnejaro you don't need to specify an env_file for `.env`, but maybe just mention that you are using it in the [README](https://github.com/geopython/pygeoapi-examples/blob/main/docker/mongo/README.md)